app/assets/javascripts/analytics/ecommerce.js

```diff
@@ -4,15 +4,17 @@
   window.GOVUK = window.GOVUK || {};
 
   var DEFAULT_LIST_TITLE = 'Site search results';
+  var DEFAULT_TRACK_CLICK_LABEL = 'Results';
 
   var Ecommerce = function (config) {
     this.init = function (element) {
       // Limiting to 100 characters to avoid noise from extra longs search queries
       // and to stop the size of the payload going over 8k limit.
-      var searchQuery   = GOVUK.analytics.stripPII(element.attr('data-search-query')).substring(0, 100).toLowerCase();
-      var ecommerceRows = element.find('[data-ecommerce-row]');
-      var startPosition = parseInt(element.data('ecommerce-start-index'), 10);
-      var listTitle     = element.data('list-title') || DEFAULT_LIST_TITLE;
+      var searchQuery     = GOVUK.analytics.stripPII(element.attr('data-search-query')).substring(0, 100).toLowerCase();
+      var ecommerceRows   = element.find('[data-ecommerce-row]');
+      var startPosition   = parseInt(element.data('ecommerce-start-index'), 10);
```

Comment:
What's different about lines 13-15? I can't see a difference - if this is a weird editor thing it'd be good to get that fixed.

Reply:
seeing that these lines were previously aligned at the length on the longest variable, i aligned them to the newest longest